src/main/java/io/strimzi/kafka/bridge/config/BridgeConfig.java

```diff
@@ -84,4 +86,8 @@ public String toString() {
                 ",httpConfig=" + this.httpConfig +
                 ")";
     }
+
+    public String getBridgeID() {
+        return config.get(BridgeConfig.BRIDGE_ID) == null ? null : config.get(BridgeConfig.BRIDGE_ID).toString();
```

Comment:
Using the ternary expression forces you into looking up the same value twice and in this case is harder to read. So better to use an if statement in this case.